### DISCUSSION TITLE
Can I add vroidcustomitem?

### DISCUSSION THREAD

**Discussion by tejasnayak25** [pixiv/three-vrm#1144] [Help]
I have an outfit saved as a vroidcustomitem file. Can I add it to my scene/character in three-vrm js?

**Comment by 0b5vr** [comment]
No, we don't support VRoid specific formats.

**Reply by tejasnayak25** [reply]
Any other methods? @0b5vr

**Reply by 0b5vr** [reply]
VRoid Studio does not support the export of each custom item afair.
I think you have to bring the VRM into some DCC like Blender to separate the item from avatars.

**Reply by tejasnayak25** [reply]
It seems that I can export the outfit texture as a png. Can I somehow add it to the character?

**Comment by tejasnayak25** [comment]
It seems that I can export the outfit texture as a png. Can I somehow add it to the character? Can I use vroidcustomitem in this way?
